engine/comms_relayer.go

```diff
@@ -82,15 +81,13 @@ func (c *commsManager) PushEvent(evt base.Event) {
 func (c *commsManager) run() {
 	defer func() {
 		// TO-DO shutdown comms connections for connected services (Slack etc)
-		atomic.CompareAndSwapInt32(&c.stopped, 1, 0)
-		atomic.CompareAndSwapInt32(&c.started, 1, 0)
 		log.Debugln(log.CommunicationMgr, "Communications manager shutdown.")
 	}()
 
 	for {
 		select {
 		case msg := <-c.relayMsg:
-			c.comms.PushEvent(msg)
+			go c.comms.PushEvent(msg)
```

Comment:
This routine doesn't seem interact with a wait group, so this doesn't add much. If this is a standardisation just disregard it.

Reply:
Nah you're right 🎉 🚗